### ISSUE TITLE
[millipede] low test coverage in some parts (Trac #1250)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1244]
The following files/directories need more test coverage:

millipede/private/millipede/MillipedeFisherMatrixCalculator.cxx
millipede/private/millipede/MuMillipede.cxx
millipede/private/millipede/TauMillipede.cxx

millipede/private/millipede/converter

Migrated from https://code.icecube.wisc.edu/ticket/1250
```json
{
    "status": "closed", 
    "changetime": "2016-02-25T20:36:43", 
    "description": "The following files/directories need more test coverage:\n\nmillipede/private/millipede/MillipedeFisherMatrixCalculator.cxx\nmillipede/private/millipede/MuMillipede.cxx\nmillipede/private/millipede/TauMillipede.cxx\n\nmillipede/private/millipede/converter", 
    "reporter": "hdembinski", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1456432603769641", 
    "component": "combo reconstruction", 
    "summary": "[millipede] low test coverage in some parts", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-20T17:12:24", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T20:59:24: 
* nega changed owner from "jbraun@icecube.wisc.edu" to "jbraun"
* nega changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160225T20:36:43: 
* jbraun changed resolution from "" to "fixed"
* jbraun commented:

> In changeset 142503:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142503"
> Tack on a fisher matrix test to pymillipede_test.py.  With additionally the MuMillipede test added by kjm, coverage is vastly improved.  Closes #1244
> ```


* jbraun changed status from "assigned" to "closed"
